pkg/bootstrapper/bootstrapper.go

```diff
@@ -48,6 +48,9 @@ const (
 	serviceWaitTime = time.Second * 10
 	// certDirectory is where the kubelet will look for certificates
 	certDirectory = "c:/var/lib/kubelet/pki/"
+	// podManifest directory where the static pod manifests should reside. This is being handled by systemd in case
+	// of linux worker nodes.
+	podManifestDirectory = "c:/k/etc/kubernetes/manifests"
```

Comment:
This needs to be relative to the install directory

Reply:
That's a good point.

Reply:
The manifest directory could be anything. I am just re-using the default dir path instead of forcing it to use a new one. While I am fine changing it, I just want to call out that if we tie this dirPath to install dir and the path defaultPath changes in future we have to change it anyways.

Reply:
I prefer not to have a proliferation of directories. 

>While I am fine changing it, I just want to call out that if we tie this dirPath to install dir and the path defaultPath changes in future we have to change it anyways.

That is not reason enough to hard code it.

Reply:
I've changed the directory to be created relative to install dir.

I agree that there shouldn't be hardcoding but that doesn't mean that we should install it from a path relative to install dir. Ideally it should be created from the ignition file especially kubelet systemd unit `[before]` section but the ignition file struct in `https://github.com/coreos/ignition/blob/spec2x/config/v2_2/types/schema.go#L203` doesn't provide us a way to access it.